helpers/lbryApi.js

```diff
@@ -23,21 +35,33 @@ module.exports = {
 			axios.post('http://localhost:5279/lbryapi', {
 				"method": "get",
 				"params": { "uri": uri, "timeout": 20}
-			}).then(function (getResponse) {
-				console.log(">> 'get' success");
-				//check to make sure the daemon didn't just time out (or otherwise send an error that appears to be a success response)
-				if (getResponse.data.result.error){
-					reject(getResponse.data.result.error);
+			}).then(function (response) {
+				console.log(">> 'get' success", response.data);
+				//check to make sure the daemon didn't just time out
+				if (response.data.result.error){
+					reject(response.data.result.error);
 				}
-				// resolve the promise with the download path for the claim we got
 				/* 
 					note: put in a check to make sure we do not resolve until the download is actually complete (response.data.completed === true)
 				*/
-				resolve(getResponse.data);
-			}).catch(function(getUriError){
+				// save a record of the file to the Files table
+				db.File.create({
+					name: response.data.result.name,
```

Comment:
This section can be simplified by avoiding repetition of `response.data.result`.